dashboard-improvements/README.md

```diff
@@ -0,0 +1,151 @@
+# Improving dashboard
+
+## Main points
+
+- Informing users about results of SRPM, COPR and Koji builds or Testing Farm
+- Replace service "Build Result" webpage with dashboard
+  - Deprecation of webpage in service
+- Database connection for dashboard
+  - Consider sensitivity of information provided (even with read-only access)
+  - Consider customizing API to the needs of dashboard
+- Consider possible improvements related to either of the points
+
+## Related issues
+
+- [Have a separate page for Testing farm](https://github.com/packit/packit-service/issues/506)
+  - Seems to be at least partially fixed: https://dashboard.stg.packit.dev/results/testing-farm/5603
+  - I guess we need to interlink all related jobs (SRPM, COPR and TF)
+- [IDs for COPR builds in API endpoint (bug)](https://github.com/packit/packit-service/issues/982)
+  - Does not affect usage of COPR build result page so far (is not used), or is the
+    reason for only one chroot result being displayed?
```

Comment:
Good, point. I am now not sure if we don't want to unify the behaviour with koji.